### ISSUE TITLE
Screen reader support missing in extensions viewlet

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#7967]
1. Turn on VoiceOver
2. Go to extensions viewlet -> whatever you do screen reader is silent
3. I would expect I hear out the text in the input box, and the currently selected extension is read out

**Comment by isidorn** [comment]
Works now
